packages/core/addon/models/user.js

```diff
@@ -10,5 +10,7 @@ export default DS.Model.extend({
   favoriteReports: DS.hasMany('report', { async: true, inverse: null }),
   deliveryRules: DS.hasMany('deliveryRule', { async: true, inverse: 'owner' }),
   dashboards: DS.hasMany('dashboard', { async: true, inverse: 'author' }),
-  favoriteDashboards: DS.hasMany('dashboard', { async: true, inverse: null })
+  favoriteDashboards: DS.hasMany('dashboard', { async: true, inverse: null }),
+  roles: DS.hasMany('role', { async: true }),
+  email: DS.attr('string')
```

Comment:
Why does the user model need an email property?

Reply:
It was for the Invite model, but I'm going to remove that model after discussing it with Jon. We're going to add the Invite model at a later stage.